src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -237,7 +237,7 @@ static string emit_string_array(const vector<string>& s, size_t max_line_length)
     return ss.str();
 }
 
-static StaticInitializers s_static_initializers;
+static StaticInitializers s_static_initializers(s_output_dir);
```

Comment:
It's not ideal to bring this stuff back in (after we got rid of it in dex) but I see why you need it. Pointing out the obvious, but all this stuff needs to be thought through and designed properly instead of ad-hoc constructs